zigzag/zigzag.py

```diff
@@ -14,7 +14,8 @@
 
 class ZigZag(object):
 
-    def __init__(self, junit_xml_file_path, qtest_api_token, qtest_project_id, qtest_test_cycle, pprint_on_fail=False):
+    def __init__(self, junit_xml_file_path, qtest_api_token, qtest_project_id,
+                 qtest_test_cycle, pprint_on_fail=False):
```

Comment:
Each argument should be on a separate line.

Reply:
That definitely looks better but I think this is one of those formatting issues that doesn't fall into pep8.

Reply:
I'm reverting it to the way it was originally, since I don't really need to change it.